AmethystAPI/src/amethyst/runtime/AmethystContext.h

```diff
@@ -15,6 +16,7 @@ class AmethystContext {
     Amethyst::EventManager mEventManager;
     Amethyst::InputManager mInputManager;
     Amethyst::PatchManager mPatchManager;
+    std::vector<Amethyst::ModInfo> mModInfo;
```

Comment:
Switch to the runtimes Mod vector

Reply:
How do you expect this to be accessible from a mod..? I thought it should be in Amethyst Context rather

Reply:
AmethstRuntime has a vector of mods, the data should be in the mod class, just love that vector in here instead

Reply:
If I would do this I would need to move the mod class into the api..
Because if I added modInfo into the runtime's mod class would make it inaccessible for mods compiled with the api.

But I think the mod class doesn't really belong in to the api, it belongs in the runtime (Nearly all of the data and functions are used by the runtime, no should ever be accessed by a mod), while the modInfo class is needed in the api to make all it's data accessible